packages/realm_dart/test/app_test.dart

```diff
@@ -5,6 +5,7 @@ import 'dart:convert';
 import 'dart:io';
 import 'dart:isolate';
 import 'package:logging/logging.dart';
+import 'package:realm_dart/src/logging.dart';
```

Comment:
This is suspicious. I would expect everything relevant to have been exported by `realm_dart`.